packages/keymaps/src/common/keymaps-validation.spec.ts

```diff
@@ -1,93 +0,0 @@
-/********************************************************************************
- * Copyright (C) 2018 Ericsson and others.
- *
- * This program and the accompanying materials are made available under the
- * terms of the Eclipse Public License v. 2.0 which is available at
- * http://www.eclipse.org/legal/epl-2.0.
- *
- * This Source Code may also be made available under the following Secondary
- * Licenses when the conditions for such availability set forth in the Eclipse
- * Public License v. 2.0 are satisfied: GNU General Public License, version 2
- * with the GNU Classpath Exception which is available at
- * https://www.gnu.org/software/classpath/license.html.
- *
- * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
- ********************************************************************************/
-import parseKeybindings from '../common/keymaps-validation';
-import { Keybinding } from '@theia/core/lib/browser';
-import * as chai from 'chai';
-
-const expect = chai.expect;
-
-describe('keymaps', () => {
-    let keybindings: Keybinding[];
-    before(() => {
-        keybindings = [];
-    });
-
-    it('parsing keybindings from well formatted raw text', () => {
-        keybindings.push(...parseKeybindings(goodContent));
-        expect(keybindings).to.have.length(2);
-    });
-
-    it('parsing keybindings from malformatted raw text', () => {
```

Comment:
Maybe we could keep the tests, as the validation schema used to be bogus and didn't match what we were expecting the `keybindings.json` file to contain. Unless you feel confident that the current and future schema will never be wrong, it might be better to keep it for "safety".